### PR TITLE
chore: allow GL pages through istio

### LINKED ISSUE #15: Ensure GitLab pages works
Likely need some work to make pages work.   

https://github.com/defenseunicorns/uds-package-gitlab/blob/main/values/gitlab-values.yaml#L129-L139

GL Pages should be working / enabled internally but aren't properly exposed through Istio.

### PR DESCRIPTION
**Pull request by Racer159**
## Description

This PR allows GL pages to work through istio in the UDS package CR

## Related Issue

Fixes #15 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Other (security config, docs update, etc)

## Checklist before merging

- [ ] Test, docs, adr added or updated as needed
- [X] [Contributor Guide Steps](https://github.com/defenseunicorns/uds-package-gitlab/blob/main/CONTRIBUTING.md#developer-workflow) followed

### CODE REVIEW

**Comment by Racer159** [comment]
Marking this as RTR for now - after thinking through testing this may be better to test in the GLR repo since pages requires GLR to operate (and otherwise we would create a circular dependency chain during testing).  For now added pages to the health check tests and here is a manual test done locally:

image

File: values/common-values.yaml [hunk 1]
```diff
@@ -1,8 +1,4 @@
 global:
-  istio:
-    enabled: true
```
Comment (zachariahmiller) [inline values/common-values.yaml:3]: ♥️

File: values/common-values.yaml [hunk 2]
```diff
@@ -130,26 +126,4 @@ gitlab:
       serviceMonitor:
         enabled: true
 
-istio:
-  enabled: false
-  injection: enabled
-  gitlab:
```
Comment (zachariahmiller) [inline values/common-values.yaml:136]: ♥️

**Review by zachariahmiller — APPROVED** [review]
:shipit:

**Pull request merged**